lib/core/services/user-preferences.service.ts

```diff
@@ -212,7 +218,14 @@ export class UserPreferencesService {
      * @returns Default locale language code
      */
     public getDefaultLocale(): string {
-        return this.appConfig.get<string>(UserPreferenceValues.Locale) || this.translate.getBrowserCultureLang() || 'en';
+        return this.appConfig.get<string>(UserPreferenceValues.Locale) || this.translate.getBrowserLang() || 'en';
```

Comment:
is that going to work with simplified Chinese and pt-BZ ? I remember we added the "getBrowserCultureLang" call to fix a few issues...

Reply:
i'll revert